.github/workflows/dev-deploy-inbound-mail.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     with:
       ee: ${{ contains (matrix.name,'ee') }}
       submodules: ${{ contains (matrix.name,'ee') }}
+      submodule_branch: "next"
```

Comment:
Should we maybe use the `current` branch here in case we do a dev deploy from a branch in the future?

Reply:
It makes sense here i will look into it.

Reply:
Now what i thought about it i am not sure, couple you provide a use case for such a case?